src/main/java/io/stargate/sgv2/jsonapi/metrics/TenantRequestMetricsFilter.java

```diff
@@ -15,7 +15,7 @@
  *
  */
 
-package io.stargate.sgv2.jsonapi.api.v1.metrics;
+package io.stargate.sgv2.jsonapi.metrics;
```

Comment:
looked into how this is used, is the reason for this bug https://github.com/stargate/data-api/issues/2023

We will need to clean this class up to make it use our new pattern, will do it after we get this merge in